core/error/error_macros.h

```diff
@@ -403,6 +403,13 @@ void _err_flush_stdout();
 	} else                                                                                             \
 		((void)0)
 
+#define ERR_FAIL_COND_STDOUT(m_cond)                                                                          \
+	if (unlikely(m_cond)) {                                                                            \
+		fprintf(stderr, "ERROR: Condition \"%s\" is true.\n   at: %s (%s:%i)\n", FUNCTION_STR, __FILE__, __LINE__, _STR(m_cond)); \
```

Comment:
```suggestion
		fprintf(stderr, "ERROR: Condition \"%s\" is true.\n   at: %s (%s:%i)\n", _STR(m_cond), FUNCTION_STR, __FILE__, __LINE__); \
```
Missed my updated comment